CIME/non_py/cprnc/README

```diff
@@ -13,14 +13,15 @@ Quick Start Guide:
 ------------------
 
 On cime supported systems you can generate a cmake Macros file using the following
-(assuming you are running the command from the directory CIME/data/cprnc):
+(assuming you are running the command from the directory CIME/non_py/cprnc):
 
 export CIMEROOT=../..
```

Comment:
I think this line (`export CIMEROOT=../..`) is incorrect, and at least on my Mac seems to be unnecessary.